src/query_results.js

```diff
@@ -73,99 +73,99 @@ module.exports = class QueryResult {
    * assemble the actual results.
    *
    * This is a recursive function, and it traverses the query graph as a tree, with
-   * every recursion passing its output queryNodeID and primaryID to the next call
+   * every recursion passing its output queryNodeID and primaryCurie to the next call
    * to use as a matching criteria for its input.
    *
    * This graphic helps to explain how this works:
    * https://github.com/biothings/BioThings_Explorer_TRAPI/issues/341#issuecomment-972140186
    *
    * The preresults returned from this method are not at all consolidated. They are
    * analogous to the collection of sets in the lower left of the graphic, which
-   * represents every valid combination of primaryIDs and kgEdgeIDs but excludes
+   * represents every valid combination of primaryCuries and recordHashs but excludes
```

Comment:
recordHashes

Reply:
To confirm....is this a decision we're discussing making? And for this PR or for a second-pass?
* ID vs Curie
* kegEdgeID vs recordHash

Reply:
I see. It's true that TRAPI uses "ID" rather than "curie" when they name stuff...